I-don't-have-Vinegrate.py

```diff
@@ -0,0 +1,68 @@
+import random
+import datetime
+import calendar
+
+"""
+This program gets from the user two dates and generates a date between the two date. If the generated
+date is Monday- the program will print 'I am out of Vinegrate', otherwise it'll print the day of that
+date.
+Note- you can only enter dates after the first unix epoch second which was at- 00:00:00 UTC on 1 January 1970
+"""
+
+# This function converts the random timestamp to a date and prints the day of the week of the
+# randomized date, or if it's day is monday- it will print 'I'm out of Vinegret!'.
+def print_solution(randed):
+    """
+    :param randed: the randed date
+    """
+    new_date = datetime.datetime.fromtimestamp(randed)
+    day = datetime.datetime.weekday(new_date)
+
+    if calendar.day_name[day] == 'Monday':
+        print("I'm out of Vinegret!")
```

Comment:
separate IO (input output) from logic in the functions, so you will be able to use them with different inputs and outputs